zapisy/apps/grade/poll/views.py

```diff
@@ -248,18 +253,19 @@ def get(self, request, semester_id=None, poll_id=None, submission_id=None):
                 self.template_name,
                 {
                     'is_grade_active': is_grade_active,
-                    'polls': group(entries=available_polls, sort=True),
+                    'polls': group(entries=available_polls, employee=request.user.employee, sort=True),
                     'results': self.__get_processed_results(submissions),
                     'results_iterator': itertools.count(),
                     'semesters': semesters,
                     'current_semester': current_semester,
                     'current_poll_id': poll_id,
-                    'current_poll': current_poll,
+                    'current_poll': current_poll.to_dict() if current_poll is not None else {},
```

Comment:
Jeśli faktycznie skończymy z metodą `to_dict` to tę ifologię będzie można przełożyć do niej, a tu zostawić samo
```suggestion
                    'current_poll': current_poll.to_dict(),
```
(Ale to trochę kwestia estetyki, tu nie będę się upierał.)